regolith/schemas.py

```diff
@@ -811,13 +814,19 @@
                                'type': 'string'},
                            'begin_year': {'required': True, 'type': 'integer'},
                            'degree': {'required': True, 'type': 'string'},
```

Comment:
On a side note, does this field coupled with begin/end date help to track the highest degree sought?

Reply:
I did think of that.  The challenge is that this field is a free-write string.  But if we could parse it for a degree from a list of degrees (BA, BSc, BS, PhD etc.) I guess it could work.    Most entries look like "PhD in Materials Science" or sthg, so it shouldn't be too hard.